openedx/features/course_duration_limits/config.py

```diff
@@ -20,6 +20,16 @@
     flag_undefined_default=False
 )
 
+# .. feature_toggle_name: global_kill_switch
+# .. feature_toggle_type: switch
+# .. feature_toggle_default: False
+# .. feature_toggle_description: Global kill switch for feature based enrollment.  To turn on the kill switch (and turn off both course duration gating and content type gating) enable a waffle flag named: 'content_type_gating.global_kill_switch'
+# .. feature_toggle_category: feature based enrollment
+# .. feature_toggle_use_cases: open_edx,
```

Comment:
Is the comma intentional?

Reply:
It was, but I also intended to write more after that. =)